app/src/features/projects/view/components/TeamMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { mdiCrown, mdiPencilOutline } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Box, Stack, Typography } from '@mui/material';
+import assert from 'assert';
+import { ProjectContext } from 'contexts/projectContext';
+import { useContext } from 'react';
+
+export interface IProjectParticipantsRoles {
+  display_name: string;
+  roles: string[];
+  initials: string;
+}
+
+const TeamMembers = () => {
+  const projectContext = useContext(ProjectContext);
+
+  // Project data must be loaded by a parent before this component is rendered
+  assert(projectContext.projectDataLoader.data);
+
+  // Define a custom sorting order for roles
+  const roleOrder: { [key: string]: number } = {
+    Coordinator: 1,
+    Collaborator: 2,
+    Observer: 3
+  };
+
+  const projectTeamMembers: IProjectParticipantsRoles[] = projectContext.projectDataLoader.data.projectData.participants
+    .map((member) => {
+      const initials = member.display_name
+        .split(',')
+        .map((name) => name.trim().slice(0, 1).toUpperCase())
+        .reverse()
+        .join('');
+      return { display_name: member.display_name, roles: member.project_role_names, initials };
```

Comment:
If we're memoizing this call, let's also return the user's avatar color in this object.